superset-frontend/packages/superset-ui-core/src/utils/featureFlags.ts

```diff
@@ -58,6 +58,7 @@ export enum FeatureFlag {
   SlackEnableAvatars = 'SLACK_ENABLE_AVATARS',
   EnableDashboardScreenshotEndpoints = 'ENABLE_DASHBOARD_SCREENSHOT_ENDPOINTS',
   EnableDashboardDownloadWebDriverScreenshot = 'ENABLE_DASHBOARD_DOWNLOAD_WEBDRIVER_SCREENSHOT',
+  TableV2TimeComparisonEnabled = 'TABLE_V2_TIME_COMPARISON_ENABLED',
```

Comment:
Do we have one for the standard one too? I think we might want to hide the Ag Grid implementation behind feature flag while it is being adopted by the community